check-with-virustotal@stilktf/files/check-with-virustotal@stilktf/check-with-virustotal@stilktf.sh

```diff
@@ -0,0 +1,18 @@
+#!/bin/bash
+
+DOMAIN="check-with-virustotal@stilktf"
+DOMAIN_DIR="${HOME}/.local/share/locale"
+_NOT_FOUND=$"Couldn't find file on VirusTotal."
+NOT_FOUND=$(/usr/bin/gettext "$_NOT_FOUND")
+
+(
+    HASH=($(sha256sum "$@"))
+    echo $HASH
+    exec 1>&-
+    JSON=$(/usr/bin/curl -X POST -H "User-Agent: VirusTotal" -H "Content-Type: application/json" -d "[{\"hash\": \"${HASH}\"}]" 'https://www.virustotal.com/partners/sysinternals/file-reports?apikey=4e3202fdbe953d628f650229af5b3eb49cd46b2d3bfe5546ae3c5fa48b554e0c' | python3 -c "import sys, json; print(json.load(sys.stdin)['data'][0]['permalink'])")
```

Comment:
This seems like it could have issues with the Public API limits if used too often. Do you have a plan in place for that?

Reply:
I just found this from GitHub - applications like OpenHashTab ([code](https://github.com/namazso/OpenHashTab/blob/455d56722b3d709ddbf1e748948d58e6bd1ae749/OpenHashTab/virustotal.cpp#L44)) uses the same endpoint and api key as the spice does

Reply:
That's akin to copying someone's password... and probably not the best practice. 😅 

Just beware of these limitations.